rna_cd/bam_process.py

```diff
@@ -0,0 +1,114 @@
+"""
+rna_cd
+Copyright (C) 2018-2019  Leiden University Medical Center, Sander Bollen
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+bam_process.py
+~~~~~~~~~~~~~~
+
+Process bam file to numpy array for classifications.
+"""
+from functools import partial
```

Comment:
I'm going to politely disagree. The latter style is imho most practical with names that can collide easily (e.g. `np.array` in stead of `array`, as `array` is also a builtin module). For most functions, however, it makes line lengths needlessly long, and I'd like to prevent needlessly line-wrapping everything: that seriously hampers readability.